docs/serving/samples/hello-world/helloworld-scala/Dockerfile

```diff
@@ -0,0 +1,22 @@
+# Use an SBT image matching the Scala and JDK version.
+FROM hseeberger/scala-sbt:8u265_1.4.2_2.12.12 as builder
```

Comment:
Do you know if there is a multi-arch image we can use here?  All of our other helloworld samples work against arm64 now 😅

Reply:
:thinking: does the build-time image matter, seeing as this is on the JVM?

Reply:
If I run the Dockerfile build on arm64 it does 😉

Reply:
He, true, quick research didn't yield anything though :/